shotover-proxy/Cargo.toml

```diff
@@ -42,6 +42,7 @@ anyhow = "1.0.31"
 
 # Parsers
 sqlparser = "0.16"
+cql3_parser = { git = "https://github.com/Claude-at-Instaclustr/rust_cql3_parser.git", branch = "main" }
```

Comment:
`main` is the default branch so we don't need to specify it. You can also remove the `.git` at the end of the URL.